src/rebar_prv_dialyzer.erl

```diff
@@ -554,21 +561,32 @@ legacy_warnings(Warnings) ->
             Warnings
     end.
 
-format_warnings(Opts, Output, Warnings) ->
+format_warnings(State, Output, Warnings) ->
+    Opts = rebar_state:opts(State),
     Warnings1 = rebar_dialyzer_format:format_warnings(Opts, Warnings),
     console_warnings(Warnings1),
-    file_warnings(Output, Warnings),
+
+    OutputFormat = get_config(State, output_format, formatted),
```

Comment:
here this could have been `rebar_opts:get(Opts, output_format, formatted)`

Reply:
I wanted to use the existing `get_config` because the output_format option is actually 1 level deeper in a proplist.
So, without using get_config, I would have to use:
```
Config = rebar_opts:get(Opts, dialyzer, []),
proplists:get_value(output_format, Config, formatted),
```

... which is very similar to what `get_config` does, but it acts on State, not Opts.

Reply:
I think that would be preferable anyway, to keep the scope of the initial function the same.